tests/foundry/lemmas.k

```diff
@@ -0,0 +1,85 @@
+requires "../contracts.k"
+
+module TEST-HELPER-LEMMAS
+  imports ASSERTTEST-CONTRACT
+
+  syntax Bytes ::= testFunction() [function]
+  syntax Set ::= testFunction2(Bytes) [function]
+
+  rule testFunction() => AssertTest.test_assert_true() [simplification]
+  rule testFunction() => #binRuntime(AssertTest) [simplification]
+  rule testFunction2(PGM) => #computeValidJumpDests(PGM) [simplification]
+
+endmodule
```

Comment:
Is this module relevant anymore?

Reply:
I think this line is still testing that we can access the `Contract.function()` syntax from a lemmas file, but I'm not sure how important that is. 
```
rule testFunction() => AssertTest.test_assert_true() [simplification]
```